knowledge-base/how-to-track-different-devices.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Capture traffic from multiple devices
+description: "Learn how to capture traffic from multiple devices and how to differentiate the traffic based on the device IP."
+type: how-to
+slug: fiddler-multiple-devices
+publish: true
+res_type: kb
+---
+
+## Environment
+
+|   |   |
+|---|---|
+| Fiddler Everywhere | 1.0.0 and above |
+| Operating System | Windows, macOS, Linux |
+
+## Description
+
+Fiddler Everywhere can simultaneously capture traffic from multiple remote computers and devices (like Android and iOS smartphones).  What configuration is needed for this to happen, and how can we differentiate the traffic from each different device?
+
+## Solution (for remote computers)
+
+**Prerequisites**
+
+- The Fiddler host and the remote computer should both be on the same network, and they should "see" each other (no firewall restrictions).
```

Comment:
Don't use "should".
See https://testdocs.telerik.com/style-guide/basic-rules-and-guidelines#use-properly-should-could-must-have-to-need-to.

Reply:
```suggestion
Before you start, verify that the host and the remote computer are on the same network, and they "see" each other (no firewall restrictions).
```